services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -398,6 +398,12 @@ func getTransformer(ctx context.Context, eslEventType db.EventType) (repository.
 	case db.EvtDeleteEnvFromApp:
 		//exhaustruct:ignore
 		return &repository.DeleteEnvFromApp{}, nil
+	case db.EvtCreateUndeployApplicationVersion:
+		//exhaustruct:ignore
+		return &repository.CreateUndeployApplicationVersion{}, nil
+	case db.EvtUndeployApplication:
+		//exhaustruct:ignore
+		return &repository.UndeployApplication{}, nil
 	default:
 		logger.FromContext(ctx).Sugar().Warnf("Found an unknown event %s. No further events will be processed.", eslEventType)
```

Comment:
I think this was the last transformer to be migrated, right?
In that case, the `default` block here should return an error and not just log a warning.